README.md

```diff
@@ -369,6 +369,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Age of Empires II](https://age-of-empires-2-api.herokuapp.com) | Get information about Age of Empires II resources | No | Yes | Unknown |
 | [AmiiboAPI](http://www.amiiboapi.com/) | Amiibo Information | No | No | Yes |
 | [Battle.net](https://dev.battle.net/) | Blizzard Entertainment | `apiKey` | Yes | Unknown |
+| [CheapShark](https://www.cheapshark.com/api) | Steam/PC Game Prices and Deals | No | Yes | Yes |
```

Comment:
Maybe `https://www.cheapshark.com` instead of `https://www.cheapshark.com/api`?

Reply:
@53jk1 - I believe the link should be to the API documentation, and not the homepage?